app/routes.js

```diff
@@ -1,7 +1,11 @@
 // const controller = require('./controllers/controller');
+const { getAlbums, getAlbumPhotos } = require('./controllers/albums');
 const { healthCheck } = require('./controllers/healthCheck');
```

Comment:
Remember you only have to declare const once in cases like this. I honestly don't mind it too much either way in terms of coding standards, its just a tip so you can write less. If you don't like it you can stick to putting `const` before each constant.